Cargo.toml

```diff
@@ -1,14 +1,55 @@
 [workspace]
-
+resolver = "2"
 members = [
     "lib/srv/desktop/rdp/rdpclient",
     "web/packages/teleport/src/ironrdp"
 ]
 
+[workspace.package]
+edition = "2021"
+license = "AGPL-3.0-only"
+homepage = "https://github.com/gravitational/teleport"
+repository = "https://github.com/gravitational/teleport"
+authors = [
+    "Teleport <goteleport.com>",
+    "Andrew Lytvynov <andrew@awly.dev>",
+    "Zac Bergquist <zac@goteleport.com>",
+    "Isaiah Becker-Mayer <isaiah@goteleport.com>",
+    "Przemko Robakowski <przemko.robakowski@goteleport.com>"
+]
+keywords = ["teleport", "rdp", "desktop"]
+publish = false
+
 [profile.dev]
 debug = 1
 lto = "off"
 
 [profile.release]
 debug = 1
 codegen-units = 1
+
+[workspace.dependencies]
+byteorder = "1.5.0"
+bytes = "1.4.0"
```

Comment:
Unless we need to keep these two (and `log`) to specific matching versions between our crates for a reason, I'd only leave the `ironrdp-` crates in here.

Reply:
db7fa28dd0c9609ae6c30e0de42086467b85a8bb